docs/getstarted/install.md

```diff
@@ -82,3 +82,48 @@ it will launch 3 components
 - mongodb UI:http://127.0.0.1:8081
 - servicecomb-kie: http://127.0.0.1:30110
 
+
+### Run natively without Docker
```

Comment:
Run natively without Docker  ----> Run locally

Reply:
Already changed it. Use `push -f`.